db/schema.rb

```diff
@@ -51,20 +67,23 @@
     t.text "wms_url"
   end
 
-  create_table "static_stats", force: :cascade do |t|
+  create_table "regional_stats", force: :cascade do |t|
```

Comment:
As this is identical to the `country_stats` (total_points is missing from the country one but we could potentially rename and use the same column both for area and points) we could define just one table for the stats and have also the `countries` table to have both countries and regions; in which case, the new combined table will also have an extra column to distinguish between countries and regions.

Another approach would be to use polymorphic associations if countries and regions are separate tables

Reply:
Hi Ferdi. I need to completely redo this CSV parser work. As I wrote it before I had the regional data, so I didn't actually know about that data. So potentially all of the PR is wrong.

Reply:
Not completely wrong but needs some reworking for sure.
By the way, the comment above was related to what we were discussing earlier about having one table only for geo entities and stats. 
Just thought about making a comment here which might help with that.